dev/TreeView/MultiLevelListViewBase.cpp

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+#include "pch.h"
+#include "common.h"
+#include "MultiLevelListViewBase.h"
+
+MultiLevelListViewBase::MultiLevelListViewBase(const ITrackerHandleManager* m_owner, const winrt::ListView& lv)
+    : m_viewModel(m_owner), m_listView(m_owner)
+{
+    m_listView.set(lv);
+}
+
+MultiLevelListViewBase::~MultiLevelListViewBase()
+{
+}
+
+com_ptr<ViewModel> MultiLevelListViewBase::ListViewModel() const
```

Comment:
>ListViewModel [](start = 43, length = 13)

maybe just ViewModel?

Reply:
(talked offline) Renaming it would create confusion because class name is also ViewModel.